README/ReleaseNotes/v636/index.md

```diff
@@ -37,6 +37,7 @@ The following people have contributed to this new version:
 * The `RooDataSet` constructors to construct a dataset from a part of an existing dataset are deprecated and will be removed in ROOT 6.38. This is to avoid interface duplication. Please use `RooAbsData::reduce()` instead, or if you need to change the weight column, use the universal constructor with the `Import()`, `Cut()`, and `WeightVar()` arguments.
 * The ROOT splash screen was removed for Linux and macOS
 * Proof support has been completely removed form RooFit and RooStats, after it was already not working anymore for several releases
+* The build options `mysql`, `odbc`, and `pgsql` have been deprecated. Please complain with root-dev@cern.ch should you still need one!
```

Comment:
Should we explicitly mention the ROOT version where it will be removed here, to be consistent with the CMake deprecation warning? (which says "will be removed in the next release of ROOT")

Reply:
In my personal opinion, we want to be flexible here: Users will only see the deprecation notice when they upgrade to 6.36. In order to give them some time to complain, we should not remove it right after the release and maybe not even for 6.38...

Reply:
I agree with @hahnjo, those may be quietly used and we don't know and if there used we may be asked by the user to either keep them forever and remove them only within a certain version-frame (timeframe)

Reply:
Ok makes sense and I agree! Just wanted to make sure you had the same thought process when making this PR

Reply:
So, can we move ahead and deprecate these three options? I have more in the pipeline that I'd like to tackle next...